src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -358,7 +358,9 @@ def _find_or_create_unresolved_symbol(location, name, scope_limit=None,
     # All requested Nodes have been checked but there has been no
     # match and there are no wildcard imports or unknown interfaces so
     # raise an exception.
-    raise SymbolError(f"No Symbol found for name '{name}'.")
+    # raise SymbolError(f"No Symbol found for name '{name}'.")
+    return symbol_table.new_symbol(
+        name, interface=UnresolvedInterface(), **kargs)
```

Comment:
I think that's the right solution (also its what _find_or_create_unresolved_interface name implies). You will need to remove the commented line and update the docstring appropriately.

Reply:
Done.